core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -394,6 +397,51 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
         return nativeImage;
     }
 
+    /**
+     * @return {@link ContainerRuntime#PODMAN} if the podman executable exists in the environment or if the docker executable
```

Comment:
The comment implies that `podman` is preferred over `docker` but the code does the opposite.

Reply:
Yeah, that was the initial behavior, but I changed later on and forgot to update the Javadoc 😛

Reply:
https://github.com/quarkusio/quarkus/pull/13699 fixes this